vtex-apps/product-summary
# Support to i18n content through CMS

#### What is the purpose of this pull request?

As the title says

#### What problem is this solving?

`Add to Cart` not becoming `Adicionar ao carrinho`, for example.

#### How should this be manually tested?

[Workspace](https://summary--storecomponents.myvtex.com)
1 - Click to edit the `ProductSummary` at storefront sidebar
2 - Edit buy button and save
3 - Switch language using admin's language switcher
4 - Content should be different for the selected language

#### Screenshots or example usage

![image](https://user-images.githubusercontent.com/15948386/58431087-9b3bc480-8082-11e9-879d-15e97252e11e.png)

![image](https://user-images.githubusercontent.com/15948386/58431117-c0c8ce00-8082-11e9-974c-d7b310bca003.png)

#### P.S.

This will be solved when vtex-apps/store-components gets up to date with these changes [here](https://github.com/vtex-apps/store-components/blob/master/react/components/DiscountBadge/index.js#L24)

![image](https://user-images.githubusercontent.com/15948386/58431133-cb836300-8082-11e9-84fe-02fbf9fcabde.png)



#### Types of changes
